pkg/arvo/sys/zuse.hoon

```diff
@@ -5570,10 +5599,11 @@
       ?~(jon ~ (some (wit jon)))
     ::                                                  ::  ++ne:dejs:format
     ++  ne                                              ::  number as real
+      =,  ^so
```

Comment:
This looks to be unused.

Reply:
It's used below, but only in one place so I removed this import.

Reply:
Ah, *that* `+so`, I was picturing `+so:dejs:format` or somesuch... Regardless, I do think the full path is better in this case.